app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split(" ")) == 3:
+        command, file, new_file = command.split(" ")
+        if command == "cp":
+            if file != new_file:
```

Comment:
There is no need to make nested `if` conditions here, write the overall condition in one `if` statement.